tests/features/chadmin_zookeeper.feature

```diff
@@ -41,3 +41,31 @@ Feature: chadmin zookeeper commands.
     """
     /test/read_sli_part/shard1/replicas/host2.net
     """
+
+
+  Scenario: Remove single host from Replicated database.
+    # Imitate that we got nodes in zk from replicated database.
+    When we execute chadmin create zk nodes on zookeeper01
```

Comment:
It's rather out of scope of this PR, but it would be great to create real replicated tables and databases rather than mock them by manually creating znodes.